src/app/app.component.ts

```diff
@@ -664,4 +664,20 @@ export class AppComponent implements OnInit, AfterViewInit {
   public duplicateRouteResponse() {
     this.environmentsService.duplicateRouteResponse();
   }
+
+  /**
+   * Enable/disable random response
+   */
+  public toggleRandomResponse(uuid: string) {
+    this.store.update(
+      toggleRandomResponse(uuid)
+    );
+  }
+
+  /**
+   * get Enable/disable random response
+   */
+  public getRandomResponse(): boolean {
+    return this.store.getActiveRoute().randomResponse;
+  }
```

Comment:
Remove as you will directly use `activeRoute` in the template.

(As a general rule, method calls should be avoided inside templates as they can have bad performances. This is mainly due to the change detection which will call the method each time something change even if the data returned by the method didn't change)